app/controllers/admin/users_controller.rb

```diff
@@ -29,20 +30,30 @@ def create
     end
   end
 
-  def confirm_delete
-    @user = User.find(params[:id])
+  def edit
+    @user.create_with_auth0
+  rescue Auth0::Exception
```

Comment:
If we rescue any `Auth0::Exception` will we know what's gone wrong in the event of an `Auth0` failure? Should we log this to PaperTrail in addition to the flash message?

Reply:
Are we logging anything else with PaperTrail? I could put it into the Rails logger maybe

Reply:
Rails logger is fine, that goes to Papertrail. We just need some reasonable string to be able to filter it, would something like `Auth0: unable to create user <exception inspection>` give us enough to go on?

Reply:
Done 👍